index.html

```diff
@@ -78,93 +124,64 @@ <h1>I'm Tarikwa <br> Glad to see you!</h1>
+        <li><a href="#"><i class="fa fa-medium" aria-hidden="true"></i></a></li>
+        </ul>  
+        <button>Get my resume</button>     
+      <section class="expertise">
+            <ul class="language">
+                <li><p>Language</p></li>
+                <li><i class="fa fa-chevron-down" aria-hidden="true"></i></li>
+            </ul>
+                <ul class="outer-item">
+                    <li class="programming"><img src="./images/javaScript.png" alt="JavaScript"></li>
+                    <li><p>JavaScript</p></li>
                 </ul>
-                    <ul class="outer-item">
-                        <li class="programming"><img src="./images/javaScript.png" alt="JavaScript"></li>
-                        <li><p>JavaScript</p></li>
-                    </ul>
-                    <ul class="outer-item">
-                        <li class="programming"><img src="./images/html.png" alt="HTML"></li>
-                        <li><p>HTML</p></li>
-                    </ul>
-                    <ul class="outer-item">
-                        <li class="programming"><img src="./images/css.png" alt="CSS"></li>
-                        <li><p>CSS</p></li>
-                    </ul>
-                <ul class="framework">
-                    <li class="flex-items"><p>Frameworks</p></li>
-                    <li><i class="fa fa-chevron-right" aria-hidden="true"></i></li>
+                <ul class="outer-item">
+                    <li class="programming"><img src="./images/html.png" alt="HTML"></li>
+                    <li><p>HTML</p></li>
                 </ul>
-                <ul class="skills">
-                    <li class="flex-items"><p>Skills</p></li>
-                    <li><i class="fa fa-chevron-right" aria-hidden="true"></i></li>
+                <ul class="outer-item">
+                    <li class="programming"><img src="./images/css.png" alt="CSS"></li>
+                    <li><p>CSS</p></li>
                 </ul>
-            </section>
-        </section>
-    </header>
+            <ul class="framework">
+                <li class="flex-items"><p>Frameworks</p></li>
+                <li><i class="fa fa-chevron-right" aria-hidden="true"></i></li>
+            </ul>
+            <ul class="skills">
+                <li class="flex-items"><p>Skills</p></li>
+                <li><i class="fa fa-chevron-right" aria-hidden="true"></i></li>
+            </ul>
+      </section>
+        <section class="contact">
+          <div class="inner-div">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          </div>
+          <form action="https://formspree.io/f/xbjwrzvy" class="form" method="post">
+            <ul class="inner-list">
+              <li><input type="text" placeholder="Enter your name" name="name"  id="name" size="40" maxlength="30"></li>
```

Comment:
- [ ] One of the requirements says that All input fields have the required attribute to activate the validation on submit. Kindly make this input required.

Reply:
I have added the required attribute to the name field.